Sources/NIO/Bootstrap.swift

```diff
@@ -146,54 +142,80 @@ public final class ServerBootstrap {
     /// - parameters:
     ///     - address: The `SocketAddress` to bind on.
     public func bind(to address: SocketAddress) -> EventLoopFuture<Channel> {
-        return bind0(eventLoopGroup: group, to: address)
+        return bind0({ address })
```

Comment:
Nit: the parentheses can be removed here.

Reply:
@aciidb0mb3r [explicitly](https://github.com/apple/swift-package-manager/pull/1536#discussion_r174348902) prefers parentheses where it suggests that the closure has no side effects. Thoughts?